interchaintest/chain_upgrade_test.go

```diff
@@ -30,9 +30,9 @@ func TestPersistenceUpgradeBasic(t *testing.T) {
 	var (
 		chainName            = "persistence"
 		upgradeRepo          = PersistenceCoreImage.Repository
-		initialVersion       = "v11.5.2"
+		initialVersion       = "v11.6.0"
```

Comment:
mainnet is bumping from `v10.4.1-fix`? 
```suggestion
		initialVersion       = "v10.4.1-fix"
```

Reply:
It's docker tag, so `v10.4.1` but you are right.